README.md

```diff
@@ -1,7 +1,7 @@
 # Flight Engine
 Mock flight data delivered simply and quickly without a database.
 
-## Developing Locally
+## Running the App Locally
 First, make sure you have [Node.js and npm](https://nodejs.org/en/download/) and [MongoDB](https://docs.mongodb.com/manual/installation/) installed, then install remaining project dependencies by running `npm install`.
```

Comment:
Since you're adding to the README can you go ahead and remove the info about MongoDB in here?